nexus/tests/integration_tests/vpc_subnets.rs

```diff
@@ -112,18 +127,24 @@ async fn test_vpc_subnets(cptestctx: &ControlPlaneTestContext) {
     assert_put_400(
         client,
         &subnet_url,
-        String::from("{ \"ipv6_block\": \"10.1.9.32/16\" }"),
+        &json!({ "ipv6_block": "10.1.9.32/16" }),
```

Comment:
Robert and I were just discussing this in an RFD PR, but we'll ultimately only allow IP subnets that use the actual network address consistent with their mask, i.e., `"10.1.0.0/16"` in this case. Not a big deal now, just wanted to flag it.

Reply:
If you have thoughts on that @davepacheco would definitely appreciate your feedback.

Reply:
That seems reasonable.  (I assume this is a bigger change than just changing this so I assume that'll be in another PR.)